### PR TITLE
Store custom by user

### LINKED ISSUE #3: Store shortcuts by user id
Other users should not be able to delete / use someone else shortcut

### PR DESCRIPTION
**Pull request by DoBrcd**
close #3

### CODE REVIEW

File: command/react/react.js [hunk 1]
```diff
@@ -12,9 +12,10 @@ const info = {
 
 const action = async (message, args) => {
   const type = args[0];
+  const id = message.author.id;
   // Merge default shortcuts with custom shortcuts
   const allShortcuts = [...defaultReact, ...shortcuts.getShortcuts()];
-  const reaction = allShortcuts.find((c) => c.name === type);
+  const reaction = allShortcuts.find((c) => c.name === type && c.userId === id);
```
Comment (Ailurus13) [inline command/react/react.js]: The filter should be done in `shortcuts.js` using lowdb !

Comment (Ailurus13) [reply command/react/react.js]: It should be the same to get the shortcut.
I think we need 2 functions in `shortcuts.js`
- getAllByUser(userId) -> Get all the shortcuts of the user
- getShortcut(userId, shortcutName) -> Get the shortcut of the user

Comment (DoBrcd) [reply command/react/react.js]: I implements getAllByUser(userID) (I just need to change the name)
I'm doing the next

Comment (DoBrcd) [reply command/react/react.js]: @CouturierTristan Take a look at the code

File: lib/shortcuts.js [hunk 2]
```diff
@@ -6,20 +6,28 @@ const FileSync = require('lowdb/adapters/FileSync');
 const adapter = new FileSync('shortcuts.json');
 const db = low(adapter);
 
-db.defaults({ shortcuts: [] }).write();
+db.defaults({ users: [] }).write();
 
-const addShortcut = (name, emojis) => {
-  if (db.get('shortcuts').find({ name }).value()) {
+const addShortcut = (userId, name, emojis) => {
+  if (!db.get('users').find({ userId }).value()) {
```
Comment (Ailurus13) [inline lib/shortcuts.js]: Is this code better ?
```suggestion
  const user = db.get('users').find({userId});
  if (user.value()) {
    [...]
  }
```

File: lib/shortcuts.js [hunk 3]
```diff
@@ -6,20 +6,28 @@ const FileSync = require('lowdb/adapters/FileSync');
...
+  }
+  if (db.get('users').find({ userId }).get('shortcuts').find({ name }).value()) {
     throw new Error(`Shortcut with name ${name} already exists`);
   }
-  db.get('shortcuts').push({ name: name.toLowerCase(), emojis }).write();
+  db.get('users').find({ userId }).get('shortcuts').push({ name: name.toLowerCase(), emojis }).write();
+};
+
+const getAllByUser = (userId) => {
+  return db.get('users').find({ userId }).get('shortcuts').value();
```
Comment (Ailurus13) [inline lib/shortcuts.js:24]: What if the user does not exists ?

**Review by Ailurus13 — CHANGES_REQUESTED** [review]

File: lib/shortcuts.js (hunk 3 above)
Comment (DoBrcd) [reply lib/shortcuts.js:24]: It will bugs

Comment (DoBrcd) [reply lib/shortcuts.js:24]: Not solve yet for me it's not a problem to work on other issues

File: lib/shortcuts.js (hunk 2 above)
Comment (DoBrcd) [reply lib/shortcuts.js]: I apply it in my last commit

**Pull request merged**